### ISSUE TITLE
Game Result Dialog show incorrect message when two players draw

### ISSUE THREAD

**Issue by HUSSAR-mtrela** [FAForever/fa#1229]
### Steps
- watch a replay with ID 4702372 or start a fight to a death between 2 ACUs
- wait for both ACUs to kill each other when one of them explodes
### Actual

Game result dialog shows message with text:

> PlayerName1 has been defeated
### Expected

Game result dialog should show message with text:

> PlayerName1 has drawn with PlayerName2
### TODO

Copy a fix in GameResults.lua from the Supreme Score Board v1.3+ mod or close this bug when SSB mod is integrated ( #1182 ) with the FAF

![image](https://cloud.githubusercontent.com/assets/11665978/14904189/7c4ca69a-0d74-11e6-81b1-98fcbf64b492.png)

**Comment by aeoncleanse** [comment]
@HUSSAR-mtrela  - I would like a working fix for this issue done for 3660 please.

**Comment by aeoncleanse** [comment]
@HUSSAR-mtrela as of 3680 is this still a thing, or is it fixed?

**Comment by HUSSAR-mtrela** [comment]
yes, it does and you can watch this replay 6216095

```INFO: Sending game result: 2 score 1
INFO: GAMERESULT : \000score 1
INFO: Sending game result: 1 score 1
INFO: GAMERESULT : \000score 1
INFO: Sending game result: 1 defeat -10
INFO: GAMERESULT : \000defeat -10
INFO: Sending game result: 2 defeat -10
INFO: GAMERESULT : \000defeat -10```

**Comment by MrRowey** [comment]
@Garanas this was fixed now correct ?

**Comment by Garanas** [comment]
Should be fixed with #4016 , if not we'll open a new issue.
